tortoise/tortoise_test.go

```diff
@@ -3053,3 +3054,20 @@ func TestUpdates(t *testing.T) {
 		require.Equal(t, id, updates[0].Blocks[0].Header.ID)
 	})
 }
+
+func TestData(t *testing.T) {
```

Comment:
I think we need some better description of what these traces are, possibly a description field at the top of the trace. The existing trace `failed_nodes.json` makes it sound like there is some failure that occurs in the trace, yet the trace is being run without error here, so what behaviour is the trace exhibiting?

Reply:
that a trace from the system test with a similar name